ExampleObjC/Podfile.lock

```diff
@@ -1,7 +1,19 @@
 PODS:
+<<<<<<< HEAD
```

Comment:
Oooooops! Git conflict!

Reply:
Forgot about the podfile in the ObjC example. I'm going to update the ObjC example app in a future PR, but thanks for pointing out! 👍